MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -571,5 +562,39 @@ public bool SupportsSubtitleConversionTo(string toCodec)
 
             return true;
         }
+
+        public (string VideoRange, string VideoRangeType) getVideoColorRange()
```

Comment:
```suggestion
        public (string VideoRange, string VideoRangeType) GetVideoColorRange()
```

Functions should begin with an uppercase letter